feat: create initial HTML layout

tisidro opened the pull request:
### Description:
Setup: SCSS Support
create layout with header, footer, and 2-column content area with a narrow left content area and wider right content area.

### Spec:
See Story: [FSA22V2-149](https://sparkbox.atlassian.net/browse/FSA22V2-149?atlOrigin=eyJpIjoiYWQ1NjQ0ODBhMGMxNDhlM2FhNjc2YzQ3NDM0MmM5YmEiLCJwIjoiaiJ9)

### Validation:

* [x] This PR has code changes, and our linters still pass.
* [x] This PR affects production code, so it was browser tested (see below).
* [ ] This PR has new code, so new tests were added or updated, and they pass.

#### To Validate:

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
2. Pull down all related branches.
3. Confirm all tests pass.

<!----- Additional validation steps below ----->

Check that index.html page layout consists of  header, footer, and 2-column content area with a narrow left content area and wider right content area.
---

#### Browser Testing
In these browsers, behavior & design closely match original specifications. A user is able to access all content and functionality, including the usability of required assistive devices, such as keyboard and screenreader.

**macOS**

* [ ] Safari (last 2 major versions)
* [x] Chrome (last 6 months)
* [ ] Firefox (last 6 months)

**Windows**

* [ ] Chrome (last 6 months)
* [ ] Firefox (last 6 months)
* [ ] Edge (last 6 months)

**Mobile**

* [ ] Safari (last 2 major versions)
* [ ] Chrome on Android (last 6 months)
* [ ] Firefox on Android (last 6 months)

tisidro committed:
feat: create initial HTML layout

create layout with header, footer, and 2-column content area with a narrow left content area and wider right content area.

FSA22V2-144

tisidro commented:
Hi Kasey, One question in case I forget: do I need to write a test for this file? Thanks! :)

kaseybon commented on scss/base.css:
```diff
@@ -0,0 +1,11 @@
+html {
```
Compiled CSS is considered an artifact and isn't committed into the repo. If you don't already have a `.gitignore` file you'll want to add one.

kaseybon commented on src/html/index.html:
```diff
@@ -1,11 +1,21 @@
-  </body>
+	<head>
+		<title>Food Moods</title>
+		<meta name="description" content="Our first page" />
```
Your meta data should be updated to match the site.

kaseybon commented on src/html/index.html:
```diff
@@ -1,11 +1,21 @@
+		<div class="page">
+			<header>header element</header>
+			<main>
+				<div class="main_wrapper">
```
I would suggest simplifying this classname a bit and just call it `class="main"`. Usually when we call something `main-wrapper` that means it's wrapping around an element with a class of `main`

kaseybon commented on src/html/index.html:
```diff
@@ -1,11 +1,21 @@
+			<header>header element</header>
+			<main>
+				<div class="main_wrapper">
+					<div class="main_left">Left Content Area</div>
```
With BEM naming convention this should be `main__left` and the line below should be `main__right`

kaseybon commented on src/scss/styles.css:
```diff
@@ -0,0 +1,36 @@
+html {
```
Should these styles be in your base.scss file?

tisidro replied:
I did try adding these to the base.scss and changing the stylesheet link to base.scss, but the styles did not load when I viewed the page.

I think I need to research how scss works to better understand why I need to move the styles to base.scss, and what else I am missing re: they are not displaying when I move them. I am going to commit the other changes for you though. I'm working Saturday so I'll spend some time figuring out how the base.scss works. Thank you! 😊

tisidro committed:
fix: update classnames, metadata, .gitignore files

update incorrect BEM class names, add metadata to reflect the site, add
css to gitignore compiled CSS.

FSA22V2-144
